src/composition.rs

```diff
@@ -887,24 +957,39 @@ mod tests {
     #[test]
     fn test_composition_string() {
         let fluxscript = r#"from(bucket: "an-composition")
-|> yield(name: "_editor_composition")
+    |> yield(name: "_editor_composition")
```

Comment:
This is a side effect of this patch that makes me perk up. Do we know why this changed?

Reply:
This is not a side effect from the patch. The spacing/identation of the input string just gets maintained in the output string.

Since I was fixing the formating of the output string (to a raw string literal), I decided to make the indentation match our standard too. I can change that back, if you want.